source/hardware/EventDrivenGP.h

```diff
@@ -147,7 +147,13 @@ namespace emp {
 
       Event & operator=(const Event &) = default;
       Event & operator=(Event &&) = default;
-
+      bool operator<(const Event & other) const {
```

Comment:
looks like we accidentally committed some of the 53vgh hacks :/

Reply:
we should get rid of these from this branch for now

Reply:
reverted by commit b7ca8e9